pkg/names/names.go

```diff
@@ -14,4 +14,8 @@ const LEADER_LABEL = "kubemacpool-leader"
 
 const LEADER_ID = "kubemacpool-election"
 
-const ADMISSION_IGNORE_LABEL = "kubemacpool/ignoreAdmission"
+const K8S_RUNLABEL = "runlevel"
+
+const OPENSHIFT_RUNLABEL = "openshift.io/run-level"
+
+var RUNLABEL = []string{"0", "1"}
```

Comment:
I still don't understand what this means. Maybe it should be named CRITICAL_RUNLABELS?